integration-cli/docker_cli_pause_test.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestEventsPause(t *testing.T) {
+	defer deleteAllContainers()
+
+	name := "testeventpause"
+	out, _, _ := dockerCmd(t, "images", "-q")
+	image := strings.Split(out, "\n")[0]
+	dockerCmd(t, "run", "-d", "--name", name, image, "sleep", "2")
```

Comment:
I know this is how it was before your edits, but its interesting that the return values aren't checked.  I'm pretty sure things will fail anyway later on, but @jfrazelle is it normal practice to check stuff like this in tests?